ProductTree/ViewModels/ProductTreeRows/ActualFiniteStateRowViewModel.cs

```diff
@@ -19,18 +19,8 @@ namespace CDP4ProductTree.ViewModels
     /// <remarks>
     /// This row shall be used when a <see cref="Parameter"/> is state dependent
     /// </remarks>
-    public class ActualFiniteStateRowViewModel : CDP4CommonView.ActualFiniteStateRowViewModel, IModelCodeRowViewModel
+    public class ActualFiniteStateRowViewModel : CDP4CommonView.ParameterBaseRowViewModel<ParameterBase>, IModelCodeRowViewModel
```

Comment:
Since its counterpart in EngineeringModel was named `ParameterStateRowViewModel` I would give this class the same name. If you decide to do so, please also rename its XAML file.

Reply:
OK, I have changed ActualFiniteStateRowViewModel into ParameterStateRowViewModel.